frontend/src/views/ReviewsView.tsx

```diff
@@ -137,12 +138,7 @@ export default function ReviewsView({
           </thead>
           <tbody>
             {reviews.map((review) => {
-              let status: string;
-              if (review.completed) {
-                status = "complete";
-              } else {
-                status = Object.keys(review.fields).length > 0 ? "draft" : "blank";
-              }
+              const status = Object.keys(review.fields).length > 0 ? "complete" : "blank";
```

Comment:
No more draft states - either "complete" (review was saved but can always be edited) or "blank" (no review entered yet)